code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -3,8 +3,7 @@
 	name = "radiation storm"
 	desc = "A cloud of intense radiation passes through the area dealing rad damage to those who are unprotected."
 
-	telegraph_duration = 400
```

Comment:
Mmm instant rad storms

Reply:
also i had to do testing again to make sure i wasn't going mental